src/System.Private.CoreLib/src/System/AppDomain.cs

```diff
@@ -284,55 +256,9 @@ private static RuntimeAssembly GetRuntimeAssembly(Assembly asm)
                 null;
         }
 
-        [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
-        private static extern void nSetNativeDllSearchDirectories(string paths);
-
-        private static void Setup(string friendlyName,
-                                    string[] propertyNames,
-                                    string[] propertyValues)
-        {
-            AppDomain ad = CurrentDomain;
-
-            if (propertyNames != null && propertyValues != null)
-            {
-                for (int i = 0; i < propertyNames.Length; i++)
-                {
-                    // We want to set native dll probing directories before any P/Invokes have a
-                    // chance to fire. The Path class, for one, has P/Invokes.
-                    if (propertyNames[i] == "NATIVE_DLL_SEARCH_DIRECTORIES")
-                    {
-                        if (propertyValues[i] == null)
-                            throw new ArgumentNullException("NATIVE_DLL_SEARCH_DIRECTORIES");
-
-                        string paths = propertyValues[i];
-                        if (paths.Length == 0)
-                            break;
-
-                        nSetNativeDllSearchDirectories(paths);
-                    }
-                }
-
-                for (int i = 0; i < propertyNames.Length; i++)
-                {
-                    if (propertyNames[i] != null)
-                    {
-                        AppContext.SetData(propertyNames[i], propertyValues[i]);
-                    }
-                }
-            }
-
-            // set up the friendly name
-            ad.nSetupFriendlyName(friendlyName);
-
-            ad.CreateAppDomainManager(); // could modify FusionStore's object
-        }
-
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        private extern void nSetupFriendlyName(string friendlyName);
 
         public int Id => GetId();
 
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        internal extern int GetId();
+        internal int GetId() => 1;
```

Comment:
The public AppDomain type with all its legacy cruft (CAS permissions, ...) lives in CoreFX.

I would like to get rid of the AppDomain.cs in CoreLib completely in next step.  I have left cleaning this one up for that since the public property will obviously disappear when I do that.